addons/l10n_gcc_invoice/models/product.py

```diff
@@ -20,4 +20,5 @@ def repl(match_occurrence):
 
         super()._compute_display_name()
         for product in self:
-            product.display_name = re.sub(r'(\d)(\s)([\u0600-\u06FF])', repl, product.display_name)
+            if isinstance(product.display_name, str):
```

Comment:
```suggestion
            if product.display_name:
```
We can simplify the condition.
The `display_name` is supposed to be a string, we just need to make sure it is defined :nerd_face: